homeassistant/util/unit_system.py

```diff
@@ -259,8 +262,13 @@ def _deprecated_unit_system(value: str) -> str:
 METRIC_SYSTEM = UnitSystem(
     _CONF_UNIT_SYSTEM_METRIC,
     accumulated_precipitation=PRECIPITATION_MILLIMETERS,
+    conversions={
+        ("distance", LENGTH_FEET): LENGTH_METERS,
+        ("distance", LENGTH_INCHES): LENGTH_CENTIMETERS,
```

Comment:
I think if we enable automatic conversion, inches -> millimeters probably makes more sense.
I expect currently it is mostly used for accumulated precipitation

```suggestion
        ("distance", LENGTH_INCHES): LENGTH_MILLIMETERS,
```